{"uuid": "083a3844-4bc8-44d2-89ef-e167abf183ec", "children": ["a6be4a37-9731-4922-b639-8f28cebc627d"], "befores": [{"name": "_Class__pytest_setup_method", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestCustomerManageAdd' object has no attribute 'go_to_customer_manage_page'\n", "trace": "  File \"D:\\softwore\\python372\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\softwore\\python372\\lib\\site-packages\\_pytest\\fixtures.py\", line 923, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\softwore\\python372\\lib\\site-packages\\_pytest\\fixtures.py\", line 778, in call_fixture_func\n    res = next(it)\n  File \"D:\\softwore\\python372\\lib\\site-packages\\_pytest\\python.py\", line 773, in xunit_setup_method_fixture\n    _call_with_optional_argument(func, method)\n  File \"D:\\softwore\\python372\\lib\\site-packages\\_pytest\\python.py\", line 691, in _call_with_optional_argument\n    func()\n  File \"D:\\my-git-project\\91lng\\selenium_91lng_auto_test\\test_case\\test_bpm\\test_customer_manage.py\", line 21, in setup_method\n    self.go_to_customer_manage_page()\n"}, "start": 1553478113988, "stop": 1553478122854}], "start": 1553478113988, "stop": 1553478122863}